xc/device.ml

```diff
@@ -1730,10 +1730,13 @@ let init_daemon ~task ~path ~args ~name ~domid ~xs ~ready_path ?ready_val ~timeo
 
 let gimtool_m = Mutex.create ()
 
-let start_vgpu ~xs task ?restore_fd domid vgpus vcpus =
+let start_vgpu ~xs task ?(restore = false) ?restore_fd domid vgpus vcpus =
 	let open Xenops_interface.Vgpu in
 	match vgpus with
 	| [{implementation = Nvidia vgpu}] ->
+		if restore then debug "start_vgpu: called with restore:true and %s restore_fd"
```

Comment:
I think this is confusing, because we'll never have `restore = True && restore_fd = None` (or the other way around).